app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (sum(split_integer(32, 6)) == 32), \
```

Comment:
Why you put sum expression in parenthesis? Also we don't use `\` for breaking lines as per our code standards - here you actually must use parenthesis